Remove Louisiana pages 186877657

vrajmohan opened the pull request.

bseeger reviewed (approved):
These changes look good. I have already removed some files in a different PR, so hopefully rebasing won't be too difficult.

vrajmohan commented:
> These changes look good. I have already removed some files in a different PR, so hopefully rebasing won't be too difficult.
I have rebased and force-pushed my PR branch.

bseeger reviewed (approved):
LGTM!
